charts/renga/values.yaml

```diff
@@ -287,16 +287,13 @@ jupyterhub:
       name: rengahub/jupyterhub-k8s
       tag: development
     services:
-      projects:
-        command: ['flask', 'run', '-p' ,'9080', '-h', '0.0.0.0']
-        url: 'http://hub:9080'
-        environment:
-          FLASK_APP: 'project_service'
-          FLASK_DEBUG: '1'
-          OAUTHLIB_INSECURE_TRANSPORT: '1'
+      notebooks:
+        url: http://renga-notebooks
         admin: true
+        api_token: notebookstoken
     extraEnv:
       DEBUG: 1
+      GITLAB_HOST: http://192.168.99.108/gitlab
```

Comment:
unset this

Reply:
ah yep, that was left on accident, thanks